blacklistfilter/ipblacklistfilter.cpp

```diff
@@ -0,0 +1,740 @@
+/**
+ * \file ipblacklistfilter.cpp
+ * \brief Module for detecting blacklisted IP addresses.
+ * \author Erik Sabik, xsabik02@stud.fit.vutbr.cz
+ * \author Roman Vrana, xvrana20@stud.fit.vutbr.cz
+ * \author Filip Suster, sustefil@fit.cvut.cz
+ * \date 2013
+ * \date 2014
+ * \date 2015
+ * \date 2018
```

Comment:
leave only 2018, or check doxygen if it is capable of handling input like date 2013 - 2018